pyteal/ast/abi/array_dynamic_test.py

```diff
@@ -1,73 +1,73 @@
 from typing import List
 import pytest
 
-from ... import *
+import pyteal as pt
 from .util import substringForDecoding
```

Comment:
We will need to address these relative imports given the new `CONTRIBUTING.md` ... but that's for a future PR